src/frontend/src/hooks/work-packages.hooks.ts

```diff
@@ -167,14 +167,14 @@ export const useAllWorkPackageTemplates = () => {
 export const useDeleteWorkPackageTemplate = () => {
   const queryClient = useQueryClient();
   return useMutation<{ message: string }, Error, string>(
-    ['work package template', 'delete'],
+    ['work package templates', 'delete'],
```

Comment:
Can u change the create worn package template funtion below these to have the s as well